project/src/main/puzzle/score-value-label.gd

```diff
@@ -17,8 +17,11 @@ func _on_resized() -> void:
 	if not _top_out_particles:
 		return
 	
+	# Avoid a Stack Overflow where changing our margins triggers another _on_resized() event, see #1810
+	disconnect("resized", self, "_on_resized")
 	margin_left = -rect_size.x
 	_top_out_particles.rect_position = rect_size * 0.5
+	connect("resized", self, "_on_resized")
```

Comment:
Maybe move this line up for symmetry with the other similar code. There is no risk of this rect_position call triggering an on_resized event.

Reply:
Moved.